sv-documentation is currently broken, so don't build it:

[ 41%] Generating showfoto/index.cache.bz2
cd /wrkdirs/usr/ports/graphics/digikam/work/digikam-5.5.0/doc-translated/sv && /usr/local/bin/meinproc5 --check --cache /wrkdirs/usr/ports/graphics/digikam/work/.build/doc-translated/sv/showfoto/index.cache.bz2 showfoto/index.docbook
index.docbook:13: element link: validity error : IDREF attribute linkend references an unknown ID "using-mainwindow-mapview"
index.docbook:30: element link: validity error : IDREF attribute linkend references an unknown ID "using-mainwindow-mapview"
kf5.kdoctools: Error: `xmllint --noout` outputted text
*** [doc-translated/sv/showfoto/index.cache.bz2] Error code 1

--- doc-translated/CMakeLists.txt.orig	2017-06-03 11:38:10 UTC
+++ doc-translated/CMakeLists.txt
@@ -1,3 +1,3 @@
 add_subdirectory( nl )
-add_subdirectory( sv )
+# add_subdirectory( sv )
 add_subdirectory( uk )
